classes/Tools.php

```diff
@@ -204,6 +204,13 @@ public static function sanitizeAdminUrl(string $url): string
         $link = Context::getContext()->link;
         if (!preg_match('@^https?://@i', $url) && $link) {
             $baseUrl = rtrim($link->getAdminBaseLink(), '/');
+
+            $physicalUri = Context::getContext()->shop->physical_uri;
```

Comment:
could you please a one-line comment to explain your change?

Reply:
> could you please a one-line comment to explain your change?

Done, is it clear enough?